File: regex.pl
#!/usr/bin/perl
use strict;
use warnings;

my $t = time;
my $match = 0;
my $nomatch = 0;

open(my $f, '<./database/fifa.csv') or die ('No hay archivo');

while(<$f>) {
    chomp;
    if(m/^([\d]{4,4})\-.*?,(.*?),(.*?),(\d+),(\d+),.*$/){
        if($5 > $4) {
			printf("%s: %s (%d) - (%d) %s\n",
				$1, $2, $4, $5, $3
			);
		}
		$match++;
    } else {
		$nomatch++;
    }
}

close($f);
printf("Se encontraron \n - %d matches\n - %d no matches\ntardo %d segundos\n", $match, $nomatch, time() - $t);

File: regex.py
#!/user/bin/python
import re

pattern = re.compile(r'^([\d]{4,4})\-\d\d\-\d\d,(.+),(.+),(\d+),(\d+),.*$')
f = open('./database/fifa.csv', 'r', encoding="utf8")

for line in f:
    res = re.match(pattern, line)
    if res:
        total = int(res.group(4)) + int(res.group(5))
        if total > 30:
            print(f"Goles: {total}, Año: {res.group(1)}, {res.group(2)} ({res.group(4)}) - ({res.group(5)}) {res.group(3)}")

f.close()